### DISCUSSION TITLE
html, css 코드 수정

### DISCUSSION THREAD

**Discussion by dubipy** [pronist/hELLO#104] [Q&A] [answered]
티스토리 편집모드에서 코드 수정하니 조금 불편하네요 혹시 해당 코드를 vscode에서 수정할 순 없을까요? 로컬에 띄우면 무한로딩되는 사이트가 뜨게되는데 혹시 방법이 있을까요?

**Comment by pronist** [comment, marked as answer]
안녕하세요 :))

티스토리 스킨 로컬에 날것으로 띄워서 하려면 개발이 수월하지 않을 수 있습니다. 아무래도 여러 코드가 뒤섞여 있다 보니 스킨이 블로그에 적용되었을 때와는 달리 예상 밖의 동작을 하기 때문입니다. hELLO 티스토리 스킨은 이 레포지토리에서 알 수 있듯이 목적코드가 아닌 소스코드를 보면 프로젝트 형태가 다르다는 것을 알 수 있는데 이는 [티도리 프레임워크](https://tidory.com)을 사용하여 개발 된 것이기 때문입니다. 

개발을 수월하게 진행하려면 티도리 프레임워크를 사용해주시고, 현재 프레임워크와는 별개로 티스토리에서 `index.xml` 가 적용되지 않는 버그가 있는데, 그 부분은 직접 `index.xml` 을 티스토리에 업로드해서 스킨 옵션이 적용될 수 있도록 해주시기 바랍니다.

**Reply by dubipy** [reply]
답변 감사합니다!!
